src/include/txn_inline.h

```diff
@@ -606,6 +606,41 @@ __wt_txn_upd_value_visible_all(WT_SESSION_IMPL *session, WT_UPDATE_VALUE *upd_va
         __wt_txn_visible_all(session, upd_value->tw.start_txn, upd_value->tw.durable_start_ts));
 }
 
+/*
+ * __wt_txn_ts_nondurable --
+ *     Is the given timestamp not yet durable? It is durable unless we are reading between the
+ *     commit time the durable time *and* the global stable timestamp has not yet advanced past the
+ *     durable time. (Note that it is not sufficient to just check that the global stable timestamp
+ *     has advanced past the commit time; prepared transactions are allowed to commit earlier than
+ *     stable if they were prepared after stable and stable has since moved up.) Assumes caller has
+ *     already checked that the commit time is visible and skips checks that makes redundant.
+ */
```

Comment:
rephrase "skips checks that makes redundant".

Reply:
Should be better now, thanks.